### PR TITLE
Updated flag "--image-pull-policy"

### PR DESCRIPTION
**Pull request by sundeepk1**
flag.StringVar(&imagePullPolicy, "image-pull-policy", "", " Image pull policy for pulling images into the cache. Possible values are 'IfNotPresent' and 'Always'. Default value is 'IfNotPresent'. Default value for Images with ':latest' tag is 'Always'")

### CODE REVIEW

File: cmd/fledged.go [hunk 1]
```diff
@@ -90,5 +90,6 @@ func init() {
...
 	flag.DurationVar(&imageCacheRefreshFrequency, "image-cache-refresh-frequency", time.Minute*15, "The image cache is refreshed periodically to ensure the cache is up to date. Setting this flag to 0s will disable refresh")
 	flag.StringVar(&dockerClientImage, "docker-client-image", "senthilrch/fledged-docker-client:latest", "The image name of the docker client. the docker client is used when deleting images during purging the cache")
-	flag.StringVar(&imagePullPolicy, "image-pull-policy", "", "Image pull policy default value is IfNotPresent. Possible values are IfNotPresent and Always")
+	flag.StringVar(&imagePullPolicy, "image-pull-policy", "", "Image pull policy default value is IfNotPresent. Possible values are IfNotPresent and Always")	
+        flag.StringVar(&imagePullPolicy, "image-pull-policy", "", " Image pull policy for pulling images into the cache. Possible values are 'IfNotPresent' and 'Always'. Default value is 'IfNotPresent'. Default value for Images with ':latest' tag is 'Always'")
```
Comment (senthilrch) [inline cmd/fledged.go]: There are two statements added. Only the second added statement is required. In the second added statement there is a space at the beginning of the description. Please remove the space

**Comment by sundeepk1** [comment]
@senthilrch is this already corrected

**Comment by senthilrch** [comment]
@sundeepk1 no, please update the PR with the changes requested.

**Comment by sundeepk1** [comment]
Hi Senthil,

My apologies for the delay, as I was in my home town with no access to internet.

I have created a new pull request for this Issue .

https://github.com/senthilrch/kube-fledged/pull/46

Please review the same.

**Comment by senthilrch** [comment]
@sundeepk1 : ok I'll close this PR and merge PR #46

**Pull request closed without merging**